policy/modules/contrib/samba.te

```diff
@@ -584,6 +591,13 @@ tunable_policy(`samba_load_libgfapi',`
     corenet_sendrecv_all_packets(smbd_t)
 ')
 
+optional_policy(`
+    tunable_policy(`samba_load_libcephfs_proxy',`
+        unconfined_stream_connect(smbd_t)
+        rw_sock_files_pattern(smbd_t, var_run_t, var_run_t)
```

Comment:
Which service creates and/or works with files with the `var_run_t` type? A private type should be used in cases of allowing write access.

Reply:
Proxy daemon, _libcephfsd_, creates the socket file for read/write under _/run_. It is a standalone daemon outside _smbd_. The proxy client library loaded by _smbd_ connects to the socket post the creation by _libcephfsd_.

Reply:
There is no service in libcephsd, which service starts/uses it?
Which filename is used?

Reply:
The catch here is that the socket file created by _/usr/sbin/libcephfsd_ can be used by any application designed to work with the proxy client library. IOW, Samba may not be the only consumer of the socket file created by _/usr/sbin/libcephfsd_ and thus it creates under _/run_.

Currently there is no service responsible for launching the daemon(_libcephfsd_). It is supposed to be running before any consumer attempts to communicate via the socket file.